lib/matplotlib/axes/_axes.py

```diff
@@ -7811,14 +7811,19 @@ def violinplot(self, dataset, positions=None, vert=True, widths=0.5,
         Make a violin plot.
 
         Make a violin plot for each column of *dataset* or each vector in
-        sequence *dataset*.  Each filled area extends to represent the
-        entire data range, with optional lines at the mean, the median,
+        sequence *dataset*.  To illustrate, if passing an 2D array/matrix, 
```

Comment:
I don't think it needs the to illustrate if you put this same information in the note.